include/aunit/reporters/aunit-reporter-junit.ads

```diff
@@ -0,0 +1,41 @@
+------------------------------------------------------------------------------
+--                                                                          --
+--                         GNAT COMPILER COMPONENTS                         --
+--                                                                          --
+--                   A U N I T . R E P O R T E R . J U N I T                --
+--                                                                          --
+--                                 S p e c                                  --
+--                                                                          --
+--                                                                          --
+--                      Copyright (C) 2020, AdaCore                         --
+--                                                                          --
+-- GNAT is free software;  you can  redistribute it  and/or modify it under --
+-- terms of the  GNU General Public License as published  by the Free Soft- --
+-- ware  Foundation;  either version 3,  or (at your option) any later ver- --
+-- sion.  GNAT is distributed in the hope that it will be useful, but WITH- --
+-- OUT ANY WARRANTY;  without even the  implied warranty of MERCHANTABILITY --
+-- or FITNESS FOR A PARTICULAR PURPOSE.                                     --
+--                                                                          --
+-- As a special exception under Section 7 of GPL version 3, you are granted --
+-- additional permissions described in the GCC Runtime Library Exception,   --
+-- version 3.1, as published by the Free Software Foundation.               --
+--                                                                          --
+-- You should have received a copy of the GNU General Public License and    --
+-- a copy of the GCC Runtime Library Exception along with this program;     --
+-- see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see    --
+-- <http://www.gnu.org/licenses/>.                                          --
+--                                                                          --
+-- GNAT is maintained by AdaCore (http://www.adacore.com)                   --
+--                                                                          --
+------------------------------------------------------------------------------
+
+--  jenkins-junit.xsd compatible reporter to file.
+package AUnit.Reporter.JUnit is
+   
+   type JUnit_Reporter is new Reporter with null record;
+      
+   procedure Report (Engine  : JUnit_Reporter;
+                     R       : Result'Class;
```

Comment:
Hello and sorry for the long wait.
We have aquired some jenkins expertise and can verify that the reporter is fine.
The only remaining thing is to change the mode of R back to "in out" now that we have rolled back this disruptive change, and we should be able to merge.